D/DistributedArrays/Compat.toml

```diff
@@ -16,5 +16,8 @@ julia = "0.6-0"
 ["0.5-0.6.0"]
 julia = "0.7-1"
 
-["0.6.1-0"]
+["0.6.1"]
 julia = "1"
+
+["0.6.2-0"]
+julia = "1.0.0-*"
```

Comment:
Keep `julia = "1"` in the project file?

Reply:
The Project file hasn't changed in that regard.

```
[compat]
julia = ">= 1.0"
```

Should it be:
```
julia = 1
```

Reply:
Yes please :) Otherwise you claim to support Julia 2.0, 3.0 etc.

> The Project file hasn't changed in that regard.

Maybe this is the first release with the new system and previous ones used the REQUIRE file?

Reply:
No the first one was https://github.com/JuliaRegistries/General/pull/426

Reply:
Ah, ok. There was some normalization turned on before.